libs/wire-api/src/Wire/API/Message/Proto.hs

```diff
@@ -168,6 +168,7 @@ toOtrRecipients =
   Msg.OtrRecipients . Client.UserClientMap
     . foldl' userEntries mempty
   where
+    -- userEntries :: Map UserId (Map ClientId Text) -> UserEntry -> Map UserId (Map ClientId Text)
```

Comment:
- [ ] remove

Reply:
This actually looks like a usefu type declaration to add to the function for improved readability to the next person looking at this.